src/core/evaluator.js

```diff
@@ -953,8 +953,9 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           });
       }
 
-      function buildTextGeometry(chars, textChunk) {
+      function buildTextGeometry(chars, textChunk, previousChars) {
```

Comment:
I could handle it below, but I felt like figuring out the `offset` made sense here, where as below it's just passing in the previous character. It doesn't have a concept of an offset. Though that is debatable, looking at the `else` down there. I'll push another commit that changes this. There's still going to end up being a check up here to make a decision on `0` or not.